addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -78,6 +92,9 @@ def _check_move_configuration(self, move):
                                   "Please create a credit note instead."))
             if line.display_type == 'product' and line.discount < 0:
                 error_message.append(_("Negative discount is not allowed, set in line %s", line.name))
+            if not line.tax_tag_ids and not any(move_line_tag.id in all_base_tags for move_line_tag in line.tax_tag_ids):
```

Comment:
@jva-odoo This is weird: 'if not line.tax_tag_ids' why do we still need the and after?

Reply:
yes, it's or let me change